python/ml_wrappers/model/openai_wrapper.py

```diff
@@ -213,6 +216,27 @@ def _call_webservice(self, data, history=None, sys_prompt=None):
                 answers.append(replace_backtick_chars(response.choices[0].message.content))
         return np.array(answers)
 
+    def _get_input_data(self, model_input, input_col):
+        if isinstance(model_input, dict):
+            prompts = pd.Series(model_input[input_col])
+            if 'history' in model_input:
+                if isinstance(model_input[input_col], str):
+                    history = pd.Series([model_input['history']])
```

Comment:
move repeating strings like 'history' to constants - either to top of file or to constants file shared across package and tests (https://github.com/microsoft/ml-wrappers/blob/main/python/ml_wrappers/common/constants.py)

Reply:
there are some examples of constants here:
https://github.com/microsoft/ml-wrappers/blob/main/python/ml_wrappers/model/openai_wrapper.py#L27

Reply:
Updated